test/parallel/test-tls-ecdh-disable.js

```diff
@@ -27,6 +27,11 @@ if (!common.hasCrypto)
 if (!common.opensslCli)
   common.skip('missing openssl-cli');
 
+const OPENSSL_VERSION_NUMBER =
+  require('crypto').constants.OPENSSL_VERSION_NUMBER;
+if (OPENSSL_VERSION_NUMBER >= 0x10100000)
+  common.skip('false ecdhCurve not supported in OpenSSL 1.1.0')
```

Comment:
missing semicolon

Reply:
Done. (In patch set that will be uploaded shortly.)